src/components/WalletConnect/NanoContract/NewNanoContractTransactionModal.js

```diff
@@ -5,66 +5,54 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, { useCallback, useEffect } from 'react';
 import {
   StyleSheet,
-  View,
   Text,
   Linking,
 } from 'react-native';
 import { t } from 'ttag';
 import { useNavigation } from '@react-navigation/native';
 import { useDispatch, useSelector } from 'react-redux';
 import { COLORS } from '../../../styles/themes';
-import { CircleInfoIcon } from '../../Icons/CircleInfo.icon';
 import { ModalBase } from '../../ModalBase';
-import SimpleButton from '../../SimpleButton';
-import { setNewNanoContractTransaction, walletConnectReject } from '../../../actions';
-import { WALLET_STATUS } from '../../../sagas/wallet';
+import { walletConnectReject } from '../../../actions';
 import { NANO_CONTRACT_INFO_URL } from '../../../constants';
+import { WarnDisclaimer } from '../WarnDisclaimer';
 
-export const NewNanoContractTransactionModal = () => {
+export const NewNanoContractTransactionModal = ({
```

Comment:
You also have the `baseStyles` here passed as props, don't you need it?

Reply:
No, this modal has been refactored to a new format created by Alex

Removed the baseStyles prop from where it's not being used 5e84654b352ae96fd25e219aed70b5c94b98ec9f


We can remove it altogether when we refactor the Connect modal to the new style